### DISCUSSION TITLE
Being notified on new versions via GitHub Announcements?

### DISCUSSION THREAD

**Discussion by ajtruckle** [MicrosoftEdge/WebView2Feedback#3966] [Q&A]
Hi

I thought I had subscribed to the announcements and yet I just found there is a new version:

![WV2](https://user-images.githubusercontent.com/7985331/180409325-1a725640-d306-4173-a4b2-3ee76fe9405e.jpg)

1. Why was I not notified?
2. Where is the changelog?

Thank you.

**Comment by victorthoang** [comment]
Hello @ajtruckle ,

I will speak with the team regarding getting notifications added in Github Announcements section. I did notice that our release page for webview2 might need to be updated, as they're normally found [here](https://docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?tabs=dotnetcsharp).

**Comment by ajtruckle** [comment]
@victorthoang 
I think there is still a delay with this? In my NuGet list it says there is a new version and I do not think it is documented?

**Comment by victorhuangwq** [comment]
Hi, we now have this repository that you can subscribed to for announcements.
https://github.com/MicrosoftEdge/WebView2Announcements
Let me know if that helps?

**Reply by sln162** [reply]
https://www.nuget.org/packages/Microsoft.Web.WebView2/
The announcement page is delayed by a few days each time, and NuGet is the fastest
